src/deleteCloudFormationStack.ts

```diff
@@ -1,4 +1,25 @@
-export const index = async (event, _context) => {
-  console.log(event);
-  console.log(_context);
+import { DynamoDBStreamEvent } from "aws-lambda";
+import { CloudFormation } from "aws-sdk";
+
+const cloudFormation = new CloudFormation();
+
+export enum Const {
```

Comment:
/nit can we give our enums a more descriptive name than `Const`?

Reply:
Done.